src/Stripe.net/Entities/_base/StripeEntity.cs

```diff
@@ -4,11 +4,46 @@ namespace Stripe
     using System.Reflection;
     using System.Runtime.CompilerServices;
     using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
     using Stripe.Infrastructure;
 
     [JsonObject(MemberSerialization.OptIn)]
     public abstract class StripeEntity : IStripeEntity
     {
+        [JsonIgnore]
+        private JObject rawJObject;
+
+        /// <summary>
+        /// Gets the raw <see cref="JObject">JObject</see> exposed by the Newtonsoft.Json library.
+        /// This can be used to access properties that are not directly exposed by Stripe's .NET
+        /// library.
+        /// </summary>
+        /// <remarks>
+        /// You should always prefer using the standard property accessors whenever possible. This
+        /// accessor is not considered fully stable and might change or be removed in future
+        /// versions.
+        /// </remarks>
+        /// <returns>The raw <see cref="JObject">JObject</see>.</returns>
+        [JsonIgnore]
+        public JObject RawJObject
+        {
+            get
+            {
+                // Lazily initialize the object the first time the getter is called.
```

Comment:
Can't we just do that at the time we parse the response to avoid having to parse twice?

Reply:
No, there is no way to parse the JSON once but deserialize it into two different objects (the proper `StripeEntity` subclass, and the raw `JObject`).